apps/ledger-live-desktop/src/storyly/components/base.tsx

```diff
@@ -0,0 +1,31 @@
+import { StorylyInstanceID } from "@ledgerhq/types-live";
+import React from "react";
+import { useStoryly } from "../useStoryly";
+
+/**
+ * Storyly Base Component's props.
+ */
+type StorylyBaseProps = {
+  instanceId: StorylyInstanceID;
+};
+
+/**
+ * Base Component for Storyly.
+ *
+ * @param instanceId
+ *
+ * @example
+ * <StorylyBase instanceId=StorylyInstanceID.testStory />
+ */
+const StorylyBase = ({ instanceId }: StorylyBaseProps) => {
+  const { ref } = useStoryly(instanceId);
+
+  return (
+    <>
+      {/* @ts-expect-error the `storyly-web` package doesn't provide any typings yet. */}
```

Comment:
I would have done that but the Storyly dev team told me that they were working on adding types to their library soon. So I figured we would be better by just doing that in the meantime rather than carrying a temporary file that'd probably be forgotten.